src/system.ts

```diff
@@ -472,7 +472,16 @@ metrics.registerQueueLengthGauge('ans104Unbundler', {
   length: () => ans104Unbundler.queueDepth(),
 });
 
-export const bundleDataImporter = new BundleDataImporter({
+export const dataImporter = new DataImporter({
```

Comment:
Minor tweak (sorry for the churn), but let's call this the `verificationDataImporter`.

Reply:
done, adjusted in the system.ts